core/store/orm/orm.go

```diff
@@ -85,19 +109,26 @@ func NewORM(uri string, timeout models.Duration, shutdownSignal gracefulpanic.Si
 	orm.MustEnsureAdvisoryLock()
 
 	db, err := initializeDatabase(string(dialect), uri)
+
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to init DB")
 	}
 
+	// FIXME: Is it desirable to set this mode in production as well?
```

Comment:
re FIXME: Not that we know of so far. I was thinking maybe we want some commands that are read only and don't require the DB lock, but even `cmd.RebroadcastTransactions` writes to the DB right now. Cut the comment?

Reply:
Well, this sets the mode in both production and test. So in order to only set it in test mode we'd need to add some additional logic or move the command.

Reply:
@j16r Do you have any thoughts?